### DISCUSSION TITLE
Request for HOOMD 2.9.7 Tutorial Access

### DISCUSSION THREAD

**Discussion by Nik2182** [glotzerlab/hoomd-blue#1697] [Support] [answered]
Hello HOOMD Community,

I am currently working with HOOMD version 2.9.7 and can't seem to make much progress. It appears that the link to the tutorial in the [official documentation for V 2.9.7](https://hoomd-blue.readthedocs.io/en/v2.9.7/) is broken or inaccessible. As version 4 of HOOMD seems to operate differently compared to 2.9.7, the latest tutorials don't quite address my needs.

Would anyone in this group happen to have access to the HOOMD 2.9.7 tutorial, or any resources related to this version, and be willing to share them? I believe such resources would be invaluable for my current project and for others who might be working with this specific version.

Any assistance or direction to relevant materials would be greatly appreciated. Thank you in advance for your time and help.

Best regards,
Nikhil

**Comment by joaander** [comment, marked as answer]
Find the 2.9.7 tutorials in the commit history of https://github.com/glotzerlab/hoomd-examples.
